sklearn/cluster/k_means_.py

```diff
@@ -792,6 +792,15 @@ class KMeans(BaseEstimator, ClusterMixin, TransformerMixin):
     inertia_ : float
         Sum of distances of samples to their closest cluster center.
 
+    See also
+    --------
+
+    MiniBatchKMeans
+        Alternative online implementation that does incremental updates
+        of the centers positions using mini-batches.
+        For large scale learning (say n_samples > 10k) MiniBatchKMeans is
+        probably much faster to than the default batch implementation.
```

Comment:
Quite a catch! I will fix it soon

-----Original Message-----
From: "Nelle Varoquaux" <notifications@github.com>
Sent: ‎7/‎22/‎2016 19:19
To: "scikit-learn/scikit-learn" <scikit-learn@noreply.github.com>
Cc: "Igor Andriushchenko" <igor.segodnya@gmail.com>; "Author" <author@noreply.github.com>
Subject: Re: [scikit-learn/scikit-learn] [MRG] DOC update refs forMiniBatchKmeans (#7057)

In sklearn/cluster/k_means_.py:
> @@ -792,6 +792,15 @@ class KMeans(BaseEstimator, ClusterMixin, TransformerMixin):
>      inertia_ : float
>          Sum of distances of samples to their closest cluster center.
>  
> +    See also
> +    --------
> +
> +    MiniBatchKMeans
> +        Alternative online implementation that does incremental updates
> +        of the centers positions using mini-batches.
> +        For large scale learning (say n_samples > 10k) MiniBatchKMeans is
> +        probably much faster to than the default batch implementation.
I think there is a extra "to" ?
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.